src/crypto/tls/tls_test.go

```diff
@@ -817,6 +817,8 @@ func TestCloneNonFuncFields(t *testing.T) {
 			f.Set(reflect.ValueOf(true))
 		case "MinVersion", "MaxVersion":
 			f.Set(reflect.ValueOf(uint16(VersionTLS12)))
+		case "SupportDelegatedCredential":
+			f.Set(reflect.ValueOf(false))
```

Comment:
I'd set this to true to check that it is properly copied. False is the default value for a bool in a struct, so you aren't actually testing the copy.

Reply:
Done.